Libraries/Components/TabBarIOS/TabBarIOS.ios.js

```diff
@@ -44,7 +55,10 @@ var styles = StyleSheet.create({
 });
 
 var config = {
-  validAttributes: ReactIOSViewAttributes.UIView,
+  validAttributes: merge(ReactIOSViewAttributes.UIView, {
```

Comment:
This is going to conflict with https://github.com/sahrens/react-native/commit/58a550fa0625f31108e06f32ebb2958e2ffefcf0#diff-27f88ff7e26d700f93b6d9791f3d408d

Can you rebase when that lands?